src/main/java/com/ebivariation/contigalias/dus/NCBIBrowser.java

```diff
@@ -0,0 +1,69 @@
+package com.ebivariation.contigalias.dus;
+
+import org.apache.commons.net.ftp.FTPFile;
+
+import java.io.IOException;
+
+public class NCBIBrowser extends FTPBrowser {
+
+    public static final String NCBI_FTP_SERVER = "ftp.ncbi.nlm.nih.gov";
+    public static final String PATH_GENOMES_ALL = "/genomes/all/";
+
+    public void connect() throws IOException {
+        super.connect(NCBI_FTP_SERVER);
+    }
+
+    public void navigateToSubDirectoryPath(String path) throws IOException {
+        super.navigateToDirectory(path);
+    }
+
+    public void navigateToAllGenomesDirectory() throws IOException {
+        navigateToSubDirectoryPath(PATH_GENOMES_ALL);
+    }
+
+    /**
+     * Takes a Genbank or Refseq accession and converts it to the equivalent path used by NCBI's FTP server.
+     * For example, on input "GCF_007608995.1" the output path is "GCF/007/608/995/GCF_007608995.1_ASM760899v1/".
+     * @param accession Any GCA or GCF String
+     * @return Path relative to ftp.ncbi.nlm.nih.gov/genomes/all/
+     * @throws IOException Passes exception thrown by FTPBrowser.listDirectories()
+     */
+    public String getGenomeReportDirectory(String accession) throws IOException {
+
+        //GCA_004051055.1
```

Comment:
It is fine for now to keep the comments as the project is just starting and I guess it is easier for you at the moment to keep them but keep in mind these very specific comments will have to be removed at some point. We do keep the ones at the header of the method (as the one you wrote)